Added reencrypt all button to workspace page

Max-7 opened the pull request:
Closes #5939 

- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description

mmmarcos reviewed (commented):
Some comments:

* In the docs, the term "re-encryption" is written with a hyphen ([Workspace-re-encryption](https://docs.parsec.cloud/en/latest/userguide/revoke_user.html#workspace-re-encryption)). Probably both are OK, but better to change it here that everywhere in the docs.
* I'm not sure about the wording: using "all" (as in "re-encrypt **all** workspaces") is misleading since not all workspaces are going to be re-encrypted. Maybe we should stick with "Re-encrypt workspaces" and having the dialog describe what would actually be re-encrypted. 
* We should update the [Workspace re-encryption](https://docs.parsec.cloud/en/latest/userguide/revoke_user.html#workspace-re-encryption) to document this feature.

vxgmichel reviewed (approved):
Ran a few tests, works fine :+1:

Max-7 commented:
> Some comments:
> 
> * In the docs, the term "re-encryption" is written with a hyphen ([Workspace-re-encryption](https://docs.parsec.cloud/en/latest/userguide/revoke_user.html#workspace-re-encryption)). Probably both are OK, but better to change it here than everywhere in the docs.
> * I'm not sure about the wording: "re-encrypt **all** workspaces" is misleading since not _all_ workspaces are going to be re-encrypted. Maybe we should stick with "Re-encrypt workspaces" and having the dialog describe what would actually be re-encrypted.
> * We should document this feature in the docs ([Workspace re-encryption](https://docs.parsec.cloud/en/latest/userguide/revoke_user.html#workspace-re-encryption)).

- Changed `reencryption` to `re-encryption`
- Changed `all workspaces` to `multiple workspaces`.
- Added the documentation. This caused diffs in the whole translation file.
